composer.json

```diff
@@ -1,10 +1,22 @@
 {
     "name": "omise/omise-magento",
-    "version": "2.3.0-dev",
+    "description": "Accept payments on your Magento 2 website with Omise",
+    "keywords": ["omise", "payment", "debitcard", "creditcard", "magento"],
+    "homepage": "https://github.com/omise/omise-magento",
+    "authors": [
+        {
+          "name": "Omise",
+          "email": "support@omise.co"
+        }
+    ],
+    "version": "2.4.0-dev",
```

Comment:
@jonrandy Omise-Magento still under `2.3.0-dev` version (we haven't released v2.3.0 yet), can you help change it back?

Reply:
Aha, yeah - oops. I'd changed that when testing to make double sure Composer pulled my new version